### PR TITLE
Define config structure for elevator screens

### PR DESCRIPTION
**Pull request by PaulJKim**
[Asana task](https://app.asana.com/0/1185117109217413/1208310396563609/f)

Defines the config structure for elevator screens.

### CODE REVIEW

**Review by digitalcora — APPROVED** [review]
~~Thanks, this looks good!~~ oops, had a comment after all

If you're feeling cautious, I'd suggest leaving this PR open for now and moving on to the next task, supporting this screen type in `screens` — then within your branch, temporarily pointing the `screens-config-lib` dependency at this branch. That will let you test that everything works together without merging. I suggest this only because I've had some problems in the past caused by a lack of "integration testing" between this library and the apps that depend on it; I don't think this particular change is risky at all. If you go ahead and merge it, the worst that can happen is that it needs a follow-up PR to fix something.

**Review by digitalcora — COMMENTED** [review]
The higher-level `Screen` definition also needs some updates to include a new app ID / screen type.

File: lib/config/screen.ex [hunk 1]
```diff
@@ -20,6 +20,7 @@ defmodule ScreensConfig.Screen do
           | :solari_large
           | :solari_large_v2
           | :pre_fare_v2
+          | :elevator_v2
```
Comment (digitalcora) [inline lib/config/screen.ex:23]: In the places in this file that currently have a list of screen types/app IDs, they are currently alphabetized. Let's keep them in order.

File: lib/config/screen.ex [hunk 2]
```diff
@@ -20,6 +20,7 @@ defmodule ScreensConfig.Screen do
...
 
   @type t :: %__MODULE__{
           vendor: :gds | :mercury | :solari | :c3ms | :outfront | :lg_mri,
```
Comment (digitalcora) [inline lib/config/screen.ex:26]: We should add the new vendor for these screens: `:mimo` (and make sure this is selectable as a vendor when adding elevator screens in the `screens` admin UI).

**Review by digitalcora — APPROVED** [review]
Couple minor comments but this looks good now!

File: lib/config/screen.ex (hunk 2 above)
Comment (digitalcora) [reply lib/config/screen.ex]: I see your latest commit says "add new vendor" but this line remains unchanged. Forgot to commit something? 🙂

Comment (PaulJKim) [reply lib/config/screen.ex]: Oops I did forget to commit 😅 thank you for catching that!

Comment (PaulJKim) [reply lib/config/screen.ex]: I also added the vendor to [line 125](https://github.com/mbta/screens-config-lib/pull/36/files#diff-126737edc2fde54ebb834201e7b8c48160e112edb8fc6428ee7be6cee6b97184L122) as assume that's a necessary change too?

**Pull request merged**